lib/services/ios-debugger-port-service.ts

```diff
@@ -103,5 +106,14 @@ export class IOSDebuggerPortService implements IIOSDebuggerPortService {
 			clearTimeout(storedData.timer);
 		}
 	}
+
+	private getTimeout(debugOptions: IDebugOptions): number {
+		let timeout = parseInt(debugOptions && debugOptions.timeout, 10);
+		if (!timeout) {
```

Comment:
Do we want to fallback to 10 secs if the user passes 0? I will expect an infinite wait if I pass 0.